modAionImpl/src/org/aion/zero/impl/config/CfgAion.java

```diff
@@ -354,7 +354,7 @@ public void toXML(final String[] args, File file) {
                     override = true;
                     String[] subArgsArr = arg.replace("--p2p=", "").split(",");
                     if (subArgsArr.length == 2) {
-                        this.getNet().getP2p().setIp(subArgsArr[0]);
+                        if (!subArgsArr[0].equals("")) this.getNet().getP2p().setIp(subArgsArr[0]);
```

Comment:
Could you run the code formatter with our formatting guidelines?  I think we put the code of the if inside { } or at least on a different line.

Reply:
will do, thanks!

Reply:
still missing the correct formatting 
```java
if (cond) {
    cmd; 
}
```